charts/m4d/values.yaml

```diff
@@ -188,4 +188,16 @@ clusterScoped: true
 
 # Set to true to enable socat in the manager pod to forward
 # traffic from a localhost registry. Used only for development.
-integrationTests: false
\ No newline at end of file
+integrationTests: false
+
+# Cluster Metadata values defined in cluster-metadata ConfigMap
+clusterMetadata:
+  # Set to the name of the cluster.
+  clusterName: hobbiton
+  # Set to cluster region.
+  region: eriador
+  # Set to cluster zone.
+  zone: theshire
+  # Set to the cluster Vault auth method path.
+  vaultAuthPath: kubernetes
```

Comment:
Is this the auth path that is expected or should this be `/v1/auth/kubernetes/login` ?

Reply:
@froesef you are right this field is passed as is to the modules values.yaml. But I prefer to leave it as is in cluster-metedata.yaml file and convert it to v1/auth/<auth path>/login in moduleinstance.go. I'll add a patch to fix that.